integrations/tensorflow/iree_tf_compiler/TFL/Passes.cpp

```diff
@@ -40,8 +40,8 @@ void buildTFLImportPassPipeline(OpPassManager &pm) {
   // Convert all TFL ops to TOSA ops
   //----------------------------------------------------------------------------
 
-  mlir::tosa::TOSATFLLegalizationPipelineOptions tosaOptions;
-  mlir::tosa::createTFLtoTOSALegalizationPipeline(pm, tosaOptions);
+  mlir::tosa::TOSATFTFLLegalizationPipelineOptions tosaOptions;
```

Comment:
Not really your change, but I think I stop being able to read after 6 all-caps that are part of different acronyms....

Reply:
I mean, this one is totally my fault. I named it but its tricky to pick on a name given the acronyms used everywhere. Happy to land as a TF+IREE google3 patch if we have a better name.

Reply:
I think I'm just complaining about LLVM style more than anything :-) Can we throw a preposition somewhere in there to break it up?

Reply:
I don't see any reason we can't though I can not think of anywhere to do that.